app/script/auth/page/Root.test.jsx

```diff
@@ -18,21 +18,37 @@
  */
 
 import React from 'react';
-import {mockStore, renderWithStore} from '../util/TestUtil';
+import {mockStore, mountWithStore} from '../util/TestUtil';
 import Root from './Root';
 
-describe('Root', () => {
-  it('renders the Wire logo', () => {
+describe('when opening the main path', () => {
```

Comment:
I thought the top level describe should always name the component your are testing.